homeassistant/components/starlink/device_tracker.py

```diff
@@ -28,7 +28,9 @@ async def async_setup_entry(
 
 
 @dataclass(frozen=True, kw_only=True)
-class StarlinkDeviceTrackerEntityDescription(EntityDescription):
+class StarlinkDeviceTrackerEntityDescription(  # pylint: disable=hass-enforce-class-module
+    EntityDescription
+):
```

Comment:
We do not currently have a `TrackerEntityDescription` - maybe it should get added in a follow-up PR?
In the meantime, I think we can ignore the violation.